src/app/main/component/events/components/event-editor/components/create-event-information/create-event-information.component.ts

```diff
@@ -70,11 +69,24 @@ export class CreateEventInformationComponent implements OnInit {
   }
 
   quillContentChanged(content: ContentChange) {
-    this.quillLength = content.text.length - 1;
-    this.isQuillUnfilled = this.quillLength < 20;
+    this.quillLength = content.text.length;
+    this.isQuillUnfilled = this.quillLength < this.minLength || this.quillLength > this.maxLength;
     this.eventInfForm.get('description').setValue(content.text.trimEnd());
   }
 
+  get getQuillLabel(): string {
```

Comment:
incorrect usage of getter. Please remove redundant get at the begining of getQuillLabel